.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
+
```

```suggestion

      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@4c0219f9ac95b02789c1075625400b2acbff50b1
        with:
          install: true

      - name: Build Docker image and store in cache
        uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825
        with:
          context: .
          push: false
          load: true
          tags: exercism/crystal-test-runner
          cache-from: type=gha
          cache-to: type=gha,mode=max

```

This will allow Docker caching to be used within `bin/run-tests-in-docker.sh`

I'll update the generic-analyzer repo to also include this bit.